lib/services/ecs/index.js

```diff
@@ -45,8 +46,8 @@ function getLatestEcsAmiId() {
 
 
 function getCompiledEcsTemplate(stackName, clusterName, ownServiceContext, ownPreDeployContext, dependenciesDeployContexts, userDataScript, ecsServiceRole) {
-    return getLatestEcsAmiId()
-        .then(latestEcsAmi => {
+    return Promise.all([getLatestEcsAmiId(), route53.listHostedZones()])
+        .then(([latestEcsAmi, hostedZones]) => {
```

Comment:
Just curious, is this an ES6 syntax that I haven't used before? I notice you're just referencing "hostedZones" by itself below, even though it appears to be the second element in an array.